src/components/ListItem/ListItem.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {
+  ListItem as MuiListItem,
+  Divider,
+  ListItemText,
+  ListItemProps as MuiListItemProps,
+  Box,
+} from '@mui/material';
+
+export interface ListItemProps extends MuiListItemProps {
+  /**
+   * The main text of list item row
+   * @default undefined
+   * @optional
+   */
+  headerText?: string;
+  /**
+   * a sorta subheader or more text to supplement main header
+   * @optional i
+   */
+  description?: string;
+  /**
+   * primary actions are passed in as children. current use cases include button, toggle, and expand.
+   * if no children is passed in, list item will display text row.
+   * @optional
+   */
+  children?: React.ReactNode;
+  /**
+   * if true, headerText will be bolded
+   * @optional
+   * @default false
+   */
+  isHeader?: boolean;
+}
+
+const ListItem = ({
+  headerText,
+  description,
+  children,
+  divider = false,
+  isHeader,
+}: ListItemProps): JSX.Element => (
+  <Box>
+    <MuiListItem
+      alignItems='flex-start'
```

Comment:
nit: double quotes (hmm I should really get prettier set up here...)

Reply:
ooh prettier will not let me do double quotes

Reply:
oh sob. another thing for the backlog then....